src/core/ApolloClient.ts

```diff
@@ -239,6 +239,7 @@ export class ApolloClient<TCacheShape> implements DataProxy {
     this.watchQuery = this.watchQuery.bind(this);
     this.query = this.query.bind(this);
     this.mutate = this.mutate.bind(this);
+    this.watchQuery = this.watchQuery.bind(this);
```

Comment:
```suggestion
    this.watchFragment = this.watchFragment.bind(this);
```

Did you mean to write this? I think this was your intention based on the changelog 🙂

Reply:
So weird! Yes, I definitely meant `watchFragment` there 🥲